.github/scripts/manage_invocation.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python3
+
+import argparse, sys, logging
+from distantrs import Invocation
+
+INVOCATION_DETAILS = '/tmp/distant-rs-invocation.txt'
```

Comment:
We need to store the invocation ID and the auth token in a well-known location (because we use this data in multiple separate steps) so I can't really see a different way to go about that. This seems to be the simplest solution to me.

Reply:
I mean the file name is ok, but the `/tmp` part is os-specific. We will plan on expanding these tests, so it would help if we aimed for portability

Reply:
Sure. I replaced it with `tempfile.gettempdir()`, supposedly it is portable.